### PR TITLE
Token border radius

### PR DESCRIPTION
**Pull request by raonivaladares**
# Description

- Creates DSBorderRadius.swift to represent token border radius.
- Adds visual representation to sample app.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

Simulator Screen Shot - iPhone 11 Pro - 2020-05-06 at 19 23 11

### CODE REVIEW

File: Tests/Core/SupportedBrands/Avon/AvonBorderRadius+Spec.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            }
+        }
+
+        describe("#large") {
+            it("returns a expect value") {
+                expect(systemUnderTest.large).to(equal(8))
+            }
+        }
+
+        describe("#standart") {
```
Comment (viviancrs) [inline Tests/Core/SupportedBrands/Avon/AvonBorderRadius+Spec.swift:28]: Acho que o describe ficou errado aqui, o método não deveria ser circle?

Comment (viviancrs) [reply Tests/Core/SupportedBrands/Avon/AvonBorderRadius+Spec.swift:28]: Replicar para Natura e TBS

Comment (raonivaladares) [reply Tests/Core/SupportedBrands/Avon/AvonBorderRadius+Spec.swift]: Falha minha, corrigido.

**Comment by raonivaladares** [comment]
Obrigado pelo review! o/

**Pull request merged**